db/migrate/20230922185815_add_devise_to_users.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+class AddDeviseToUsers < ActiveRecord::Migration[7.0]
+  def self.up
+    change_table :users do |t|
+      ## Database authenticatable
+      t.string :email,              null: false, default: ""
+      t.string :encrypted_password, null: false, default: ""
+
+      ## Recoverable
+      t.string   :reset_password_token
+      t.datetime :reset_password_sent_at
+
+      ## Rememberable
+      t.datetime :remember_created_at
```

Comment:
Guys everything looks great but are we going to use devise like this? Because what I understand here is that it creates columns at the database for password and reset_password, columns that we are not going to use at the project, right? Maybe we can talk about this on monday so me and @Alejuda can understand a little bit more about this 👍